lms/validation/_api.py

```diff
@@ -12,15 +12,6 @@ class APIRecordSpeedgraderSchema(JSONPyramidRequestSchema):
     document_url = fields.Str()
     """URL of the document for this assignment."""
 
-    canvas_file_id = fields.Str()
```

Comment:
These are set here:

https://github.com/hypothesis/lms/blob/simplify-speed-grader-config/lms/resources/_js_config/__init__.py#L386

Only document_url is there.